build.gradle

```diff
@@ -473,28 +473,30 @@ task reportCoverage(dependsOn: ['jacocoRootReport', 'core:reportCoverage'])
 for ( sv in availableScalaVersions ) {
   String taskSuffix = sv.replaceAll("\\.", "_")
 
-  tasks.create(name: "jar_core_${taskSuffix}", type: GradleBuild) {
+  tasks.create(name: "jar_scala_${taskSuffix}", type: GradleBuild) {
     startParameter = project.getGradle().getStartParameter().newInstance()
     startParameter.projectProperties += [scalaVersion: "${sv}"]
-    tasks = ['core:jar']
+    tasks = ['core:jar', 'streams:streams-scala:jar']
   }
 
-  tasks.create(name: "test_core_${taskSuffix}", type: GradleBuild) {
+  tasks.create(name: "test_scala_${taskSuffix}", type: GradleBuild) {
     startParameter = project.getGradle().getStartParameter().newInstance()
     startParameter.projectProperties += [scalaVersion: "${sv}"]
-    tasks = ['core:test']
+    tasks = ['core:test', 'streams:streams-scala:test']
   }
 
   tasks.create(name: "srcJar_${taskSuffix}", type: GradleBuild) {
     startParameter = project.getGradle().getStartParameter().newInstance()
     startParameter.projectProperties += [scalaVersion: "${sv}"]
+    // core has a separate source jar for each scala version, streams-scala source is included
+    // in the single streams sources jar
```

Comment:
@guozhangwang @mjsax The behaviour with regards to the Scala source mentioned in the comment above is a bit weird.

Reply:
I think we have a separate jar for each scala version as well for kafka-streams-scala?